src/main/java/baritone/command/defaults/SetChestCommand.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of Baritone.
+ *
+ * Baritone is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Baritone is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package baritone.command.defaults;
+
+import baritone.api.IBaritone;
+import baritone.api.cache.IWaypoint;
+import baritone.api.cache.Waypoint;
+import baritone.api.command.Command;
+import baritone.api.command.argument.IArgConsumer;
+import baritone.api.command.exception.CommandException;
+import baritone.api.utils.BetterBlockPos;
+import net.minecraft.block.Block;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class SetChestCommand extends Command {
+
+    public SetChestCommand(IBaritone baritone) { super(baritone, "setchest"); }
+
+    @Override
+    public void execute(String label, IArgConsumer args) throws CommandException {
+        args.requireMax(0);
+        Optional<BlockPos> blockPos = ctx.getSelectedBlock();
+
+        int x = blockPos.get().getX();
+        int y = blockPos.get().getY();
+        int z = blockPos.get().getZ();
+
+        if (blockPos.isPresent()) {
+            Block block = ctx.world().getBlockState(blockPos.get()).getBlock();
+            if (block.equals(Blocks.CHEST) || block.equals(Blocks.ENDER_CHEST) || block.equals(Blocks.TRAPPED_CHEST)) {
```

Comment:
This will screw up on any other ChestBlock besides these three. Any reason why you chose not to use an instanceOf ChestBlock check here instead to preserve mod support?

Reply:
there is no reason. I just didn't have modded chests in mind